pipelines/DarkEnergyCamera/ProcessCcd.yaml

```diff
@@ -11,6 +11,7 @@ tasks:
   calibrate:
     class: lsst.pipe.tasks.calibrate.CalibrateTask
     config:
+      file: $AP_PIPE_DIR/config/DECam/calibrate.py
       photoCal.match.referenceSelection.magLimit.fluxField: 'i_flux'
```

Comment:
As mentioned in the PR description, I think it's better *not* to try integrating the configs for now, since that will make it harder to tell what came from where (this is especially true for the configs that need to be copied from the base pipeline). I could add a comment saying "do not merge"?

Reply:
Yeah, adding a comment here is a good idea.